client/src/routes/+layout.svelte

```diff
@@ -177,6 +192,11 @@
     left: 8px;
   }
 
+  h3 {
+    margin-top: 10px;
+    margin-bottom: 10px;
```

Comment:
These can be combined into margin-block
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block

Reply:
Done